src/main/java/uk/nhs/tis/trainee/usermanagement/api/UserAccountResource.java

```diff
@@ -129,4 +140,70 @@ ResponseEntity<Void> deleteCognitoAccount(@PathVariable String username) {
     log.info("Deleted Cognito account for user '{}'.", username);
     return ResponseEntity.noContent().build();
   }
+
+  /**
+   * List assigned groups of the given user.
+   *
+   * @param username The username of the user.
+   * @return 204 No Content, if successful.
+   */
+  @GetMapping("/user-groups/{username}")
```

Comment:
I still think we want to move these to a separate resource, `/api/user-account/user-groups` and `/api/user-account/dsp-consultants/enroll` doesn't feel quite right vs `/api/user-groups` and `/api/user-groups/dsp-consultants/enroll`

Reply:
Nice shout. It is refactored with the latest commit. Thanks @Judge40.